src/controller/controller.c

```diff
@@ -423,6 +423,25 @@ bool controller_apply_config(Controller *controller) {
                 if (selinux_context && !node_set_required_selinux_context(node, selinux_context)) {
                         return false;
                 }
+
+                const char *proxy_enabled_nodes = cfg_s_get_value(
```

Comment:
Wouldn't it be better to extract this from cfg only once, outside of the loop?

Reply:
Its done for each node section `[node <name>]` to extract node-specific configuration, so it actually has to be done inside the loop.

Reply:
Right! Missed that part.